crt_portal/static/js/attachments.js

```diff
@@ -0,0 +1,17 @@
+(function(root, dom) {
+  var add_attachment_el = document.getElementById('add_attachment');
+  var file_el = document.getElementById('file_input');
+
+  add_attachment_el.onclick = function(event) {
+    event.preventDefault();
+    file_el.click();
+  };
+
+  file_el.onchange = function(event) {
+    var form = document.getElementById('complaint-view-attachments');
+    form.submit();
+
+    var modal = document.getElementById('attachment-uploading--modal');
+    root.CRT.openModal(modal);
```

Comment:
This opens the modal but generates an exception on modal.js:50 when `openModal` attempts to set the focus to the first focusable element. A byproduct of openModal being implemented for modals with interactions though in this use-case the modal has none.

Reply:
Thanks! Added a hidden input to the attachment modal